BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -490,6 +490,84 @@ RegisterContainer (
   return Status;
 }
 
+/**
+  This function would check container validity.
+
+  @param[in]  Signature   Container signature
+
+  @retval  BOOLEAN        Container valid/not valid
+
+**/
+BOOLEAN
+IsContainerValid (

```

Comment:
We should not use this to validation container. 
The signature could be anything.   Not limited to the ones we defined here.

Reply:
There is no standard way to check. For checking container, we need to register it, access components and its contents. Accessing non-container image w.r.r container/component headers, it causes out of bound access.
If CFGD is given as input to container registration, currently it succeeds. 
Could we handle this check at platform level as it known available containers in flash map?

Reply:
Updated this functionality using available functionalities